apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -355,6 +360,20 @@ public void log(Request request, Response response) {
             HTTP_ACCESS_RESPONSE_OUTPUT_SIZE_IN_BYTES,
             String.valueOf(response.getHttpOutput().getWritten()));
 
+        String responseDurationInMilliseconds =
```

Comment:
Is this guaranteed to return a string? If not, we should at least check for null or wrap in try block.